patches/components-content_settings-core-common-content_settings_mojom_traits.cc.patch

```diff
@@ -1,10 +1,12 @@
 diff --git a/components/content_settings/core/common/content_settings_mojom_traits.cc b/components/content_settings/core/common/content_settings_mojom_traits.cc
-index 84eceacac92268b43302fed7ce3358e1aa4b5adb..863e78557ba5cb2fd91a9c7596921d4aa3288c13 100644
+index 7db7683e5a8544f29772c2676a4298230494ab53..863e78557ba5cb2fd91a9c7596921d4aa3288c13 100644
 --- a/components/content_settings/core/common/content_settings_mojom_traits.cc
 +++ b/components/content_settings/core/common/content_settings_mojom_traits.cc
-@@ -101,6 +101,8 @@ bool StructTraits<content_settings::mojom::RendererContentSettingRulesDataView,
+@@ -99,7 +99,10 @@ bool StructTraits<content_settings::mojom::RendererContentSettingRulesDataView,
+          RendererContentSettingRules* out) {
+   return data.ReadImageRules(&out->image_rules) &&
           data.ReadScriptRules(&out->script_rules) &&
-          data.ReadAutoplayRules(&out->autoplay_rules) &&
++         data.ReadAutoplayRules(&out->autoplay_rules) &&
```

Comment:
these should all be converted to a `define` so we can reduce the patch to one line